CHANGELOG.md

```diff
@@ -4,6 +4,482 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.21.0](https://github.com/go-gitea/gitea/releases/tag/1.21.0) - 2023-11-12
+
+* BREAKING
+  * Restrict certificate type for builtin SSH server (#26789)
+  * Refactor to use urfave/cli/v2 (#25959)
+  * Move public asset files to the proper directory (#25907)
+  * Remove commit status running and warning to align GitHub (#25839)
```

Comment:
Warning was added back so changelog should be changed to reflect that only running was removed

Reply:
I don't think it's necessary, the revert PR is part of the changelog, and the blog article only mentions running, not warning.

Reply:
But if you look at changelog especially as this information is in different sections this is very misleading. I would say they should be compressed into the single changelog entry with reference to both PR

Reply:
Okay, why not actually - done :)